statsmodels/survey/tests/tests_summary_stats.py

```diff
@@ -0,0 +1,19 @@
+from summary_stats import SurveyStat
+import unittest
+import pandas as pd
+import numpy as np
+
+df = pd.read_csv("examples/survey_df.csv")
+data = df.copy()
+n, p = data.shape
+svy = SurveyStat(df, cluster="dnum", prob_weights="pw")
+
+class TestSummary(unittest.TestCase):
+    def test_intialization(self):
+        np.testing.assert_equal(data["dnum"].reshape(n,1), svy.cluster)
+        np.testing.assert_equal(data["pw"].reshape(n, 1), svy.prob_weights)
+        self.assertEqual(None, svy.strata)
+
+
+suite = unittest.TestLoader().loadTestsFromTestCase(TestSummary)
```

Comment:
we just use nosetests which is much easier, and we will switch to pytest in future,
So we don't need the `unittest` module
see #2232 for how to selectively run specific test modules or specific unit tests